letter-shop.md

```diff
@@ -0,0 +1,232 @@
+# Letter Shop Tutorial
+
+## What you need before you start:
+
+* a laptop with an internet connection and NodeJS installed
+* basic knowledge of the command line terminal
+* basic knowledge of JavaScript, including Promises and Buffers
+* (optional) the concept of one-way hash functions
+
+## What you'll learn:
+
+* the concept Hash Time Lock Agreements (HTLAs)
+* the Ledger Plugin Interface (LPI)
+* how to build a paid web app using Interledger!
+
+## Step 1:
+
+Save the following JavaScript as `shop.js`:
+
+```js
+const http = require('http')
```

Comment:
What about using something like koa instead? I think it's pretty uncommon in the Node ecosystem to use the http module directly

Reply:
hm, interesting argument. I don't think I've ever met a NodeJS developer who was unfamiliar with the http module though, have you? And I definitely know there are people the other way around, for instant backend developers (including myself) would be more familiar with the http module than with koa. Also, in the second tutorial I need to flush the headers, so I think it wouldn't work there.